pkg/protocols/http/raw/raw.go

```diff
@@ -97,6 +97,13 @@ func Parse(request string, inputURL *urlutil.URL, unsafe, disablePathAutomerge b
 	default:
 		cloned := inputURL.Clone()
 		cloned.Params.IncludeEquals = true
```

Comment:
not related to this PR , but sharing here for context , since we will need to remove IncludeEquals hack we use now

- https://github.com/projectdiscovery/nuclei/issues/4963
- https://github.com/projectdiscovery/utils/issues/379